test/download.spec.ts

```diff
@@ -1,19 +1,30 @@
 import { expect, use } from "chai";
 import Chaifs = require("chai-fs");
 import path = require("path");
+import { configurationToExecute } from "./utils";
 
 use(Chaifs);
 
-describe("Media Assets", () => {
-  it("should create the assets directory", () => {
-    expect(path.join(__dirname, "out/assets")).to.exist;
-  });
+const interactions = configurationToExecute();
+
+interactions.forEach(interactionFile => {
+  describe("Media Assets", async () => {
+    before(async function() {
+      if (interactionFile.interactionFileName !== "interaction-google.json") {
```

Comment:
could we perhaps add the name of the interaction to the `describe` ? That way we know which test is that we're skipping from the output, also, seems like for this specific case, we shouldn't just skip the test, but actually not declare it at all when on xlsx, that is, it's just not supported on that case